azure-pipelines.yml

```diff
@@ -0,0 +1,78 @@
+pr: [master]
+
+trigger: [master]
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  - group: fluent-secrets
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
+
+  - script: |
+      yarn
+    displayName: Install dependencies
+
+  - script: |
+      yarn lint
+    displayName: Lint
+
+  - script: |
+      yarn prettier
+    displayName: Prettier
+
+  - script: |
+      yarn test
+    displayName: Unit tests
+
+  - script: |
+      bash <(curl -s https://codecov.io/bash)
+    displayName: Report coverage
+
+  - script: |
+      yarn test:e2e
+    displayName: E2E tests
+
+  - script: |
+      yarn test:projects
+    displayName: Project tests
+
+  - script: |
+      yarn test:circulars
+    displayName: Circular dependency tests
+
+  - script: |
+      if [ $BUILD_SOURCEBRANCHNAME == 'master' ]; then
+        yarn stats:build
+      else
+        echo 'skip bundle statistics collection'
+      fi
+    displayName: Bundle statistics (master only)
+
+  - script: |
+      if [ $BUILD_SOURCEBRANCHNAME == 'master' ]; then yarn perf; fi
+    displayName: Performance tests (master only)
+
+  - script: |
+      if [ $BUILD_SOURCEBRANCHNAME == 'master' ] && [ -n "${STATS_URI}" ]; then
+        yarn stats:save --tag=`git tag --points-at HEAD`
```

Comment:
`stats:save` reads several env variables, not sure all are available under the same name in azure.
See https://github.com/microsoft/fluent-ui-react/blob/master/build/gulp/tasks/stats.ts#L158-L167

Reply:
Great call out... we'll have to map those to these: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables

Reply:
Thanks for catching that! I'll update it to check both sets of variables for now.

Reply:
Should be fixed now